src/unstable/components/Form/story.js

```diff
@@ -32,9 +34,11 @@ import * as ReactDOM from 'react-dom'
 import { Form } from 'rendition/dist/unstable/'
 import { MermaidWidget } from 'rendition/dist/extra/Form/mermaid'
 import { MarkdownWidget } from 'rendition/dist/extra/Form/markdown'
+import { MarkdownEditorWidget } from 'rendition/dist/extra/Form/markdownEditor'
 
 // Register the extra format widgets to the Form component
 Form.registerWidget('markdown', MarkdownWidget)
+Form.registerWidget('markdownEditor', MarkdownEditorWidget)
```

Comment:
I don't think we need another format here, this editor should replace the existing `markdown` format